torch/_dynamo/trace_rules.py

```diff
@@ -3294,6 +3296,7 @@ def _module_dir(m: types.ModuleType):
     "torch._higher_order_ops.invoke_subgraph",
     "torch._higher_order_ops.scan",
     "torch._higher_order_ops.strict_mode",
+    "torch._higher_order_ops.triton_kernel_wrap",
```

Comment:
If we don't allow this function here in trace_rules, then do_prune_configs will get marked as a "SkipFunctionVariable" instead of a UserFunctionVariable---https://github.com/pytorch/pytorch/blob/5e67bea7aa3e256243020e10978bd869d62472ae/torch/_dynamo/trace_rules.py#L71